apps/web/app/s/[surveyId]/components/LinkSurvey.tsx

```diff
@@ -137,7 +137,18 @@ export default function LinkSurvey({
           formbricksSignature={product.formbricksSignature}
           onDisplay={async () => {
             if (!isPreview) {
-              const { id } = await createDisplay({ surveyId: survey.id }, webAppUrl);
+              const api = new FormbricksAPI({
+                apiHost: webAppUrl,
+                environmentId: "",
```

Comment:
@ShubhamPalriwala Not setting the environmentId looks very dangerous. Please make the environmentId mandatory and always set it :-)